apps-rendering/src/components/layout/standard.stories.tsx

```diff
@@ -0,0 +1,77 @@
+// ----- Imports ----- //
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import { breakpoints } from '@guardian/source-foundations';
+import { partition } from '@guardian/types';
+import { withKnobs } from '@storybook/addon-knobs';
+import Standard from 'components/layout/standard';
+import { article, matchReport, review } from 'fixtures/item';
+import { renderAll } from 'renderer';
+
+// ----- Stories ----- //
+
+const display = ArticleDisplay.Standard;
+
+export const Article = (): React.ReactNode => {
+	return (
+		<Standard item={article}>
+			{renderAll(
+				{
+					theme: ArticlePillar.News,
+					design: ArticleDesign.Standard,
+					display,
+				},
+				partition(article.body).oks,
+			)}
+		</Standard>
+	);
+};
+Article.story = { name: 'Article' };
+
+export const Review = (): React.ReactNode => {
+	return (
+		<Standard item={review}>
+			{renderAll(
+				{
+					theme: ArticlePillar.Culture,
+					design: ArticleDesign.Standard,
```

Comment:
I think this should be `ArticleDesign.Review`?

I suspect this still renders the stars because `review` has `design === ArticleDesign.Review`. Possibly the API for the layouts could be improved to prevent the possibility of this mismatch 🤔